core/server/api_container/server/startosis_engine/kurtosis_types/directory/directory.go

```diff
@@ -0,0 +1,115 @@
+package directory
+
+import (
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/builtin_argument"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/kurtosis_type_constructor"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/startosis_errors"
+	"go.starlark.net/starlark"
+)
+
+const (
+	DirectoryTypeName = "Directory"
+
+	ArtifactNameAttr  = "artifact_name"
+	PersistentKeyAttr = "persistent_key"
+	DescriptionAttr   = "desc"
```

Comment:
is it planned to use this constant later? because now it's not used

Reply:
Ah yes, it was intended to be used but we don't now. I'll remove, thanks for flagging!